src/openrct2/Context.h

```diff
@@ -1,4 +1,4 @@
-﻿/*****************************************************************************
+/*****************************************************************************
```

Comment:
BOM?

Reply:
Most likely yeah, must've happened automatically. Probably a good idea to normalize them in another PR.

Reply:
Indeed.

For this PR, can you revert it? The PR otherwise looks good to me.

Reply:
Will do, fixing the tests right now.

Reply:
The last commit reintroduces this change again, done by Visual Studio. Will fix it after CI is done with this commit.